### PR TITLE
cross_val changes

### PR DESCRIPTION
**Pull request by rishabhfyi**
I am not sure which is the best approach, commenting out the previous code or removing it. Thus, commented as of now.
Also, kept self._training_features and self._training_labels assigned as these are being used in functions in other python files, which are being called here.

### CODE REVIEW

File: few/few.py [hunk 1]
```diff
@@ -213,7 +215,7 @@ def fit(self, features, labels):
         initial_estimator = copy.deepcopy(self.ml.fit(x_t,y_t))
         # self._best_estimator = copy.deepcopy(self.ml.fit(x_t,y_t))
 
-        self._best_score = self.ml.score(x_v,y_v)
+        self._best_score = np.amax(cross_val_score(self.ml,x_t,y_t,cv=3))
```
Comment (lacava) [inline few/few.py]: use mean

File: few/few.py [hunk 2]
```diff
@@ -329,9 +331,10 @@ def fit(self, features, labels):
             try:
                 # if len(self.valid_loc(F)) > 0:
                 if self.valid_loc():
-                    tmp_score = self.ml.score(self.transform(
-                                    x_v,self.pop.individuals)[:,self.valid_loc()],
-                                    y_v)
+                    #tmp_score = self.ml.score(self.transform(
+                    #                x_v,self.pop.individuals)[:,self.valid_loc()],
+                    #                y_v)
+                    tmp_score = np.amax(cross_val_score(self.ml,x_t,y_t,cv=3))
```
Comment (lacava) [inline few/few.py]: use mean

**Review by lacava — CHANGES_REQUESTED** [review]
i think we can get rid of self._training_features. it is a legacy holdover and isn't used anymore.

File: few/few.py (hunk 1 above)
Comment (lacava) [inline few/few.py]: I don't think you need to have cv=3 in there, that is the default

Comment (RGBuilds) [reply few/few.py]: yeah, realized that later..will do

File: few/few.py (hunk 2 above)
Comment (RGBuilds) [reply few/few.py]: will do

File: few/few.py (hunk 1 above)
Comment (RGBuilds) [reply few/few.py]: will do

**Comment by rgbuilds** [comment]
self._training_features is used in population.py in line 235 which is called in few.py line 50 

few.py line 50 
self.pop = self.init_pop(x_t.shape[0])

population.py line 235
for i,p in it.zip_longest(
                         range(self._training_features.shape[1]),
                         pop.individuals,fillvalue=None):

**Comment by lacava** [comment]
ok, let's leave _training_features in for now then.

**Pull request merged**